src/krux/pages/home_pages/wallet_descriptor.py

```diff
@@ -39,41 +39,41 @@ def __init__(self, ctx):
         self.ctx = ctx
 
     def wallet(self):
-        """Handler for the 'wallet' menu item"""
+        """Handler for the 'wallet descriptor' menu item"""
+        title = t("Wallet output descriptor")
         self.ctx.display.clear()
         if not self.ctx.wallet.is_loaded():
             text = t("Wallet output descriptor not found.")
-            if not self.ctx.wallet.is_multisig():
-                text += " " + t("It is optional for single-sig.")
             self.ctx.display.draw_centered_text(text)
             if self.prompt(t("Load one?"), BOTTOM_PROMPT_LINE):
-                return self._load_wallet()
+                self._load_wallet()
```

Comment:
removing the return here introduced a bug alongside with the other change on the line with the comment `# Try to save the Wallet output descriptor on the SD card`

Reply:
The bug is, if you don't load the wallet output descriptor or load an with the error `Invalid wallet:`, it will still asks the user to save the file on the SD card

Reply:
Thank you!